src/OrchardCore.Modules/OrchardCore.Shortcodes/Startup.cs

```diff
@@ -124,4 +128,18 @@ public override void ConfigureServices(IServiceCollection services)
             });
         }
     }
+
+    [RequireFeatures("OrchardCore.Deployment", "OrchardCore.Shortcodes.Templates")]
+    public class ShortcodeTemplatesDeployementStartup : StartupBase
+    {
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.AddRecipeExecutionStep<ShortcodeTemplateStep>();
```

Comment:
This line goes in the normal startup, as Recipes are their own module (not part of Deployment) and recipes are AlwaysOn

Reply:
I think it should go to `ShortcodeTemplatesStartup` while it requires `ShortcodeTemplateManager`

Reply:
That's what I said